src/Pages/Signup-Page/signup.css

```diff
@@ -0,0 +1,119 @@
+.signup-page {
+  font-family: "Montserrat", sans-serif;
+  display: grid;
+  grid-template-rows: 4rem minmax(calc(100vh - 9rem), 1fr) 5rem;
+  grid-template-areas:
+    "header"
+    "main"
+    "footer";
+}
+
+.signup__main {
+  padding-bottom: 4rem;
+  grid-area: main;
+  background-color: var(--primary-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* signup form Stats Here */
+
+.signup__title {
+  border-bottom: 2px solid var(--primary-color);
+  padding: 0.5rem;
+  text-align: center;
+  margin-bottom: 1rem;
+  color: var(--primary-color);
+  font-size: var(--font-size-X-large);
+}
+.signup-form {
+  margin: 0.5rem;
+  display: flex;
+  flex-direction: column;
+  padding: 1rem 1.5rem;
+  border: 2px solid var(--white-color);
+  border-radius: 5px;
+  box-shadow: 0px 0px 8px 1px rgb(199, 198, 198);
+  background-color: var(--white-color);
+}
+.signup-form__input--usergmail {
+  max-width: 23rem;
+}
+.signup-form__label {
+  color: var(--primary-color);
+  font-size: var(--font-size-regular);
+  font-weight: 600;
+}
+.signup-form--password-wrapper {
+  position: relative;
+}
+.signup-form__input--password {
+  width: 100%;
+}
+.signup-form__input--password-hide-icon {
+  position: absolute;
+  right: 5px;
+  top: 45%;
+}
+.signup-form--name-wrapper,
+.signup-form--password-wrapper {
+  display: flex;
+  flex-direction: column;
```

Comment:
Suggestion: You can create CSS utility classes for spacing (margin, padding), flex, etc. this way CSS file will be smaller and more maintainable